### PR TITLE
suggest a Python version

### PR DESCRIPTION
**Pull request by theaverageguy**
Fixes https://github.com/sunpy/sunpy/issues/1704

### CODE REVIEW

File: doc/source/guide/installation/index.rst [hunk 1]
```diff
@@ -20,7 +20,9 @@ Installing Scientific Python and SunPy
 If you do not currently have a working scientific Python distribution this
 guide will set you up with the Anaconda Python distribution. Alternative options
 exist for different operating systems and can be found later in the
-:ref:`advanced-install` section.
+:ref:`advanced-install` section.Sunpy currently supports Python 2.7 with Anaconda.
+The documentation is designed for Python 2.7. Support for Python 3.x would be
```
Comment (wafels) [inline doc/source/guide/installation/index.rst]: I'd suggest changing the the Python 3.x sentence to "Support for Python 3.x is coming soon".

Comment (theaverageguy) [inline doc/source/guide/installation/index.rst]: Sure @wafels. :)

**Comment by theaverageguy** [comment]
Take a look @wafels . :)

**Comment by ayshih** [comment]
I appreciate that you're addressing this issue, but I feel that your addition – while technically true – is buried in the section about installing Anaconda.  Someone might already have a non-Anaconda Python 3.x environment, and it might not be clear to them whether or not that is compatible with SunPy.

I suggest an alternate approach of two separate additions:

* Near the top, saying that we support Python 2.7, with 3.x support coming soon
* In the sentence where we direct them to download the Anaconda installer, clarifying _which_ installer to download (namely the 2.7 one)

**Comment by Cadair** [comment]
closing due to python 3 support being merged.

**Pull request closed without merging**